tests/string/RGB.test.ts

```diff
@@ -0,0 +1,106 @@
+import * as RA from 'fp-ts/ReadonlyArray'
+import * as E from 'fp-ts/Either'
+import { pipe, tuple } from 'fp-ts/function'
+import * as RGB from '../../src/string/RGB'
+import { cat, combineExpected, validateArbitrary } from '../../test-utils'
+
+const valid: ReadonlyArray<string> = [
+  'rgb(0,0,0)',
+  'rgb(255,255,255)',
+  'rgba(0,0,0,0)',
+  'rgba(255,255,255,1)',
+  'rgba(255,255,255,.1)',
+  'rgba(255,255,255,0.1)',
+  'rgb(5%,5%,5%)',
+  'rgba(5%,5%,5%,.3)',
+]
+
+const invalid: ReadonlyArray<string> = [
+  'rgb(0,0,0,)',
+  'rgb(0,0,)',
+  'rgb(0,0,256)',
+  'rgb()',
+  'rgba(0,0,0)',
+  'rgba(255,255,255,2)',
+  'rgba(255,255,255,.12)',
```

Comment:
🤔 I'm a little bit surprised that this is invalid. I guess css is more strict than JS about leading zeroes in floats?

Reply:
per spec it's not. see: https://w3c.github.io/csswg-drafts/css-values-4/#number-value

tried it in chrome devtools and it works fine